client/src/components/Instructions/index.js

```diff
@@ -117,7 +211,7 @@ export default class Instructions extends Component {
 
           <div className={styles.step}>
             <div className={styles.instruction}>
-              b. Visit the tutorials. Start with the <a href='/counter'> Counter</a> page to deploy and interact with your first contract.
+              b. Visit the tutorials. Start with the <a href='/counter'> Counter</a> page to deploy and interact with your first contract, or deploy a <a href='/gasless-counter'>Gasless Counter</a> which utilizes metatransactions.
```

Comment:
Should we add a link in the top navbar as well, next to the existing Counter, or within a "Tutorials" dropdown? The reference here is easy to miss.

Reply:
Yes, my plan was to add this when we were ready to make the tutorial public - I will add it now to the navbar.

Reply:
Added